### PR TITLE
Refactor Room Page

### PR DESCRIPTION
**Pull request by YuliaFursovych**

### CODE REVIEW

**Review by Blager89 — APPROVED** [review]

File: client/src/components/RoomPage/RoomPage.js [hunk 1]
```diff
@@ -4,53 +4,116 @@ import "react-responsive-carousel/lib/styles/carousel.min.css";
...
-const photoSet = [
+const images = [
     {
-        src: "http://lorempixel.com/1600/1100/nature/6/",
-        width: 1600,
-        height: 1100
+        src: "https://c3.staticflickr.com/9/8583/28354353794_9f2d08d8c0_b.jpg",
+        thumbnail: "https://c3.staticflickr.com/9/8583/28354353794_9f2d08d8c0_n.jpg",
+        thumbnailWidth: 320,
+        thumbnailHeight: 174,
```
Comment (yodeco) [inline client/src/components/RoomPage/RoomPage.js]: we will not be able to receive width/height from server for image as I undersand. Roompage will be displayed ok without width/height?

File: client/src/components/RoomPage/RoomPage.js [hunk 2]
```diff
@@ -4,53 +4,116 @@ import "react-responsive-carousel/lib/styles/carousel.min.css";
...
+        thumbnail: "https://steemitimages.com/DQmbsy3neKZAtKHCR4z8DR9zU5Ae6m6chjn2HVYMVRc8sxg/pho.jpg",
+        thumbnailWidth: 612,
+        thumbnailHeight: 320,
+        tags: [{value: "Nature", title: "Nature"}, {value: "People", title: "People"}],
+    }
 ];
+
 function TabContainer(props) {
     return (
         <Typography component="div" style={{ padding: 8 * 3 }}>
```
Comment (yodeco) [inline client/src/components/RoomPage/RoomPage.js]: What means syntax {{ padding: 8 * 3 }} ? And why {{ - double, not single? Never met it before, it is some material ui special syntax?

**Review by yodeco — APPROVED** [review]

File: client/src/components/RoomPage/RoomPage.js (hunk 2 above)
Comment (YuliaFursovych) [reply client/src/components/RoomPage/RoomPage.js]: Yes, it's from material ui, but I've moved it to a style file.

File: client/src/components/RoomPage/RoomPage.js (hunk 1 above)
Comment (YuliaFursovych) [reply client/src/components/RoomPage/RoomPage.js]: Yes, it will be displayed ok without width/height.

**Pull request merged**